window.go

```diff
@@ -21,8 +21,14 @@ type Window struct {
 	fixedSize  bool
 
 	onClose        func(Event) bool
-	onKeyDown      func(Event) bool
 	onScreenResize func(Event)
+
+	onKeyDown *keyDownCb
+}
+
+type keyDownCb struct {
+	data interface{}
```

Comment:
data is used by the callback user, i.e:

```go
type UserData struct {
...
}
myData := &UserData{}
wnd.wnd.OnKeyDown(func(evt Event, data interface{}) bool{
     md := data.(UserData)
     // do some data manipulation here this could be some key handling logic i.e alt+<key>
    // manipulation where the first hit detects alt then do some special handling after that
   // and so on
}, myData)

```